packages/webpack/lib/shims/server.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-require('source-map-support/register');
+if (!require('module').prototype._compile.__sourceMapSupport) {
```

Comment:
This looks a bit fishy, isn't this handled by the package already? https://github.com/evanw/node-source-map-support/blob/master/source-map-support.js#L17

Reply:
True, that does look bad - and it ought to be handled by the package. I initially thought that the reason might be that we bundle *all the things*, but apparently, it is not. It want(ed) to get this fix shipped, so I went with the solution we have now.